Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+gem 'rails', '~> 7.0.0'
+
```

Comment:
I have a slight preference for keeping the gems in alpha order but would not block over this. Unless you found that declaring this dependency above the others was required for some reason. I wouldn't be too surprised...

Reply:
That's where Jcoyne put it. Let me try moving and see what happens.

Reply:
kk, appreciate it